guides/custom_video.md

```diff
@@ -0,0 +1,215 @@
+
+```
+    // Switch the camera
+    cameraCaptureSource.switchCamera()
+
+    // Get the current device and format
+    let currentDevice = cameraCaptureSource.device
+    let currentFormat = cameraCaptureSource.format
+
+    // Pick a new device explicitly 
+    let newDevice = MediaDevice.listVideoDevices().first { mediaDevice in
+        mediaDevice.type == MediaDeviceType.videoFrontCamera
+    }
+    cameraCaptureSource.device = newDevice
+
+    // Pick a new format explicitly (reverse these so the highest resolutions are first)
+    let newFormat = MediaDevice.listSupportedVideoCaptureFormats(mediaDevice: videoDevice).reversed().filter { $0.height < 800 }.first
+    if let format = newFormat {
+        cameraCaptureSource.format = format
+    }
+```
+
+5. To turn on the flashlight on the current camera, set [CameraCaptureSource.torchEnabled](https://aws.github.io/amazon-chime-sdk-ios/Protocols/CameraCaptureSource.html#/c:@M@AmazonChimeSDK@objc(pl)CameraCaptureSource(py)torchEnabled). This can be set before or after capture has been started, and before or during call.
+
+```
+    // Turn on the torch
+    cameraCaptureSource.torchEnabled = true
+
+    // Turn off the torch
+    cameraCaptureSource.torchEnabled = false
+```
+
+6. To render local camera feeds before joining the call, use [VideoSource.addVideoSink](https://aws.github.io/amazon-chime-sdk-ios/Protocols/VideoSource.html#/c:@M@AmazonChimeSDK@objc(pl)VideoSource(im)addVideoSinkWithSink:) with a provided [VideoSink](https://aws.github.io/amazon-chime-sdk-ios/Protocols/VideoSink.html) (e.g. a [DefaultVideoRenderView](https://aws.github.io/amazon-chime-sdk-ios/Classes/DefaultVideoRenderView.html) created as described in [Getting Started](https://github.com/aws/amazon-chime-sdk-ios/blob/master/guides/getting_started.md#render-a-video-tile)).
+```
+    // Add the render view as a sink to camera capture source
+    cameraCaptureSource.addVideoSink(sink: someDefaultVideoRenderView)
+```
+
+To use the capture source in a call, do the following:
+
+1. When enabling local video, call [AudioVideoControllerFacade.startLocalVideo(source:)](https://aws.github.io/amazon-chime-sdk-ios/Protocols/AudioVideoControllerFacade.html#/c:@M@AmazonChimeSDK@objc(pl)AudioVideoControllerFacade(im)startLocalVideoWithSource:) with the camera capture source as the parameter. Ensure that the capture source is started to start transmitting frames.
```

Comment:
Do we want to explicitly mention "before calling `startLocalVideoWithSource `" in last sentence?